modules/core/src/main/java/org/locationtech/jts/geom/CoordinateSequence.java

```diff
@@ -105,15 +105,16 @@ default int getMeasures() {
   default boolean hasZ() {
       return (getDimension()-getMeasures()) > 2;
   }
-  
+
   /**
-   * Checks {@link #getMeasures()} to determine if {@link #getM(int)}
-   * is supported.
-   * 
+   * Tests whether the coordinates in the sequence have measures associated with them. Returns true
+   * if {@link #getMeasures()} > 0. See {@link #getMeasures()} to determine the number of measures
```

Comment:
Actually can also use @see Javadoc for this

Reply:
@dr-jts : It seems like `@see` does not cope with text after?  So I think `@link` might be best.  If you have a way that `@see` will work, can you tell me what it is?

https://docs.oracle.com/javase/7/docs/technotes/tools/windows/javadoc.html#see